compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Output.hs

```diff
@@ -43,6 +43,7 @@ instance RenderDoc ModuleDoc where
         , section "Templates" md_templates
         , section "Template Instances" md_templateInstances
         , section "Typeclasses" md_classes
+        , section "Typeclass Instances" (filter id_isOrphan md_instances)
```

Comment:
Should we make the title `Orphan Typeclass Instances`? Otherwise it seems somewhat confusing that only those are show here.

Reply:
👍  sounds good